src/alfrescoApi.ts

```diff
@@ -73,17 +75,34 @@ export class AlfrescoApi implements Emitter, AlfrescoApiType {
         this.clientsFactory();
 
         this.processClient = new ProcessClient(this.config, this.httpClient);
+        this.basicApiClient = new AlfrescoApiClient(undefined, this.httpClient);
 
         this.errorListeners();
-        this.initAuth(config);
+        if(this.oauthInit){
+            this.initAuth(config);
 
-        if(this.isLoggedIn()){
-            this.emitBuffer('logged-in');
+            if(this.isLoggedIn()){
+                this.emitBuffer('logged-in');
+            }
         }
 
         return config;
     }
 
+    public callCustomApiWithoutAuth(...rest: Parameters<typeof this.basicApiClient.callCustomApi>) {
```

Comment:
Why do we have a client here?

Reply:
Because in ADF we shouldn't call the `callCustomApi` method using the `oauth2Auth` instanciated by `AlfrescoApi` 
https://github.com/Alfresco/alfresco-ng2-components/pull/8344/files#diff-b2a9de8ac310fdd7e634be4947fe8939b6d61f6f74d5d55474617b815a8d7e58R43
so we've exposed the `callCustomApi` method provided by the `AlfrescoApiClient`